### PR TITLE
Get bin collection date for given address (Monash Council)

### PR DESCRIPTION
**Pull request by ARMAGEDDONAPOCALYPSE**
Get bin collection date for given address (Monash Council)

### CODE REVIEW

File: WifiController.py [hunk 1]
```diff
@@ -26,16 +61,19 @@ def scanNetwork(self):
...
+        print("------------------------------------------------------------------------------------------------")
         print("Connecting to WIFI: %s" % self.SSID)
         wifi.radio.connect(self.SSID, self.Password)
-
         print("Connected! My IP address is: ", wifi.radio.ipv4_address)
 
+    def stop_access_point(self):
+        wifi.radio.stop_ap()
+
     def setDateTime(self, timezone_offset):
```
Comment (morpha1337) [inline WifiController.py:72]: def setDateTime(self, timezone_offset) -> struct_time

File: Bindicator.py [hunk 2]
```diff
@@ -41,11 +67,19 @@ def startProgram():
...
     ##Light Sleep Untill Notification Dismissal or Date Change.
 
+def bin_color_display(key):
+    global gbit
+    if(key == 'Landfill Waste'):
+        gbit.top(GlowBitController.RED)
+    else:
+        gbit.top(GlowBitController.YELLOW)
+    gbit.bottom(GlowBitController.GREEN)
+
```
Comment (morpha1337) [inline Bindicator.py:84]: bin_color_display() should be in Monash.py

File: Bindicator.py [hunk 3]
```diff
@@ -7,27 +7,53 @@
 
 import json
 import Monash
+import BlinkPatterns
+
+address = '3/36 Panorama Street, Clayton 3168' #Replace your address to get geolocationid (Monash Council only for now)
+gbit = GlowBitController.GlowBitController(config["glowbit"]) #Global gbit
```
Comment (morpha1337) [inline Bindicator.py:13]: Why did you make this global?

Comment (ARMAGEDDONAPOCALYPSE) [reply Bindicator.py:13]: gbit is used in another func

File: Bindicator.py [hunk 4]
```diff
@@ -42,10 +68,19 @@ def startProgram():
     ##Connect to wifi and set Date Time
     #wifi.connect()
     wifi.setDateTime(10)
+
     ##Get Bin Data
     ##Check Against Date
     ##Light Sleep Untill Notification Dismissal or Date Change.
 
+def bin_color_display(key):
```
Comment (morpha1337) [inline Bindicator.py:76]: Please keep Monash specific code in the Monash File. Different council areas have different Bin Color!

File: Bindicator.py (hunk 3 above)
Comment (morpha1337) [reply Bindicator.py:13]: please move that Function to Monash.py and put this back into startDebug()

**Pull request closed without merging**